src/reducers/versions.tsx

```diff
@@ -1037,21 +1076,31 @@ export const goToRelativeDiff = ({
   pathList,
   position,
   versionId,
+  comparedToVersionId,
 }: GoToRelativeDiffParams): ThunkActionCreator => {
   return async (dispatch, getState) => {
     const { router, versions: versionsState } = getState();
     const version = getVersionInfo(versionsState, versionId);
+    const entryStatusMap = getEntryStatusMap(
+      versionsState,
+      versionId,
+      comparedToVersionId,
+    );
 
     if (!version) {
       throw new Error('Cannot go to relative diff without a version loaded.');
     }
+    if (!entryStatusMap) {
+      throw new Error('Cannot get status map.');
```

Comment:
It would be nice for debugging to put `versionId` and `comparedToVersionId` in this error message.